supplementary_style_guide/glossary_terms_conventions/general_conventions/d.adoc

```diff
@@ -160,6 +176,16 @@ Always spell out in full and capitalize as shown here, unless part of a command.
 
 *See also*: xref:debug-adj[debug (adjective)]
 
+[[deduplication]]
+==== image:images/yes.png[yes] deduplication (noun)
+*Description*: A feature of virtual disk optimization (VDO) that helps conserve storage space by eliminating multiple copies of duplicate blocks. Instead of writing the same data more than once, VDO detects blocks that are duplicates of an existing block, and records them as a reference to the original block. VDO maintains a mapping from logical block addresses, which are used by the storage layer above VDO, to physical block addresses, which are used by the storage layer under VDO. After deduplication, multiple logical block addresses can be mapped to the same physical block address; these are called shared blocks. Block sharing is invisible to users of the storage, who read and write blocks as they would if VDO were not present. When a shared block is overwritten, a new physical block is allocated for storing the new block data to ensure that other logical block addresses that are mapped to the shared physical block are not modified. Use "block" or "VDO" or "local storage device" instead of using the vague term "device".
```

Comment:
```suggestion
*Description*: _Deduplication_ is a feature of virtual disk optimization (VDO) that helps conserve storage space by eliminating multiple copies of duplicate blocks. Instead of writing the same data more than once, VDO detects blocks that are duplicates of an existing block, and records them as a reference to the original block. VDO maintains a mapping from logical block addresses, which are used by the storage layer above VDO, to physical block addresses, which are used by the storage layer under VDO. After deduplication, multiple logical block addresses can be mapped to the same physical block address; these are called shared blocks. Block sharing is invisible to users of the storage, who read and write blocks as they would if VDO were not present. When a shared block is overwritten, a new physical block is allocated for storing the new block data to ensure that other logical block addresses that are mapped to the shared physical block are not modified. Use _block_ or _VDO_ or _local storage device_ instead of using the vague term _device_.
```
- Not sure who the `who` is in this sentence: `Block sharing is invisible to users of the storage, who read and write blocks as they would if VDO were not present`.
- On a related note, is this level of detail necessary here? It's much more detailed than other entries. Maybe consider cutting everything except the first and last sentences or finding some other method of paring this content down? Just a thought.